### DISCUSSION TITLE
high sensitive FC Mode for sniffing near by transmissions

### DISCUSSION THREAD

**Discussion by mentanah** [egzumer/uv-k5-firmware-custom#311] [Ideas]
Not really sure how the frequency and CTCSS copy mode works in background. But would it be possible to increase sensitivity and speed to sniff used frequency of radio transmissions from nearby devices like pmr headsets etc?

**Comment by gartnl** [comment]
That is imo not possible. In this mode the HT works as any frequency counter. It just displays any strong signal. If you where to add gain you'd have a problem. A nearby ( let's say 50 meter ) PMR signal strength is lower than that of a FM broadcast station. You just would see the broadcast station. If at all, because the amplification causes all signals to be stronger and the counter would not now what to display.

However, if you now in what range ( eg PMR ) you expect the signal you could setup the spectrum analyser to see what channel it is on.
And after that you could use the ctss scan to get that setting too.

**Reply by mentanah** [reply]
Ok, that makes sense. Thank you for this explanation!
